app/views/admin/index.html.erb

```diff
@@ -4,6 +4,30 @@
   <%= notice %>
```

Comment:
I am not sure but I think we need to remove that file as well. We do have the `/admin/welcome/index` page now